README.md

```diff
@@ -106,12 +106,12 @@ Peripheral send `unsync` if has different state.
 c) begin rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w
 p) unsync rnbqkbnr/pppppppp/8/8/8/P7/1PPPPPPP/RNBQKBNR b
 ```
-Peripheral can send even `w` (white), `b` (black), `?` (unknown) instead of full piece information depending on internal sensors and knowledge.
+Peripheral can send even `u` (unknown piece) or `?` (unknown piece and color) instead of full piece information depending on internal sensors and knowledge.
```

Comment:
Peripheral can send  `u` (unknown white piece), `U` (unknown black) or `?` (unknown piece and color) instead of full piece information depending on internal sensors and knowledge.

Reply:
Peripheral can send `u` (unknown black), `U` (unknown white) or `?` (unknown) instead of full piece information depending on internal sensors and knowledge.
?

Reply:
it is ok